pkg/ovs/openflow/interfaces.go

```diff
@@ -170,6 +170,17 @@ type CTAction interface {
 	LoadToMark(value uint32) CTAction
 	LoadToLabelRange(value uint64, rng *Range) CTAction
 	MoveToLabel(fromName string, fromRng, labelRng *Range) CTAction
+	// NAT action is used if the packet is not committed into the conntrack zone, and is required to leverage the
```

Comment:
Could you explain a little more on NAT()? Did not understand it after reading the comment.